srcpkgs/zeroc-mcpp/patches/Makefile-install.diff

```diff
@@ -0,0 +1,31 @@
+diff --git a/Makefile b/Makefile
+index bfbf543..8938583 100644
+--- a/Makefile
++++ b/Makefile
+@@ -3,6 +3,7 @@
+ #
+ 
+ PREFIX ?= /opt/mcpp-2.7.2
++DESTDIR = 
+ 
+ ifeq ($(CFLAGS),)
+ 	CFLAGS = -O2
+@@ -55,9 +56,16 @@ $(LIBDIR)/libmcpp.a: $(OBJS)
+ 	$(AR) rcs $(LIBDIR)/libmcpp.a $(OBJS)
+ 
+ install: $(LIBDIR)/libmcpp.a
+-	@mkdir -p $(PREFIX)/$(LIBDIR)
+-	cp $(LIBDIR)/libmcpp.a $(PREFIX)/$(LIBDIR)
++	@mkdir -p $(DESTDIR)$(PREFIX)/$(LIBDIR)
++	cp $(LIBDIR)/libmcpp.a $(DESTDIR)$(PREFIX)/$(LIBDIR)
++	chmod 755 $(DESTDIR)$(PREFIX)/$(LIBDIR)/libmcpp.a
++	@mkdir -p $(DESTDIR)$(PREFIX)/include
++	cp mcpp_lib.h $(DESTDIR)$(PREFIX)/include
++	chmod 644 $(DESTDIR)$(PREFIX)/include/mcpp_lib.h
++	cp mcpp_out.h $(DESTDIR)$(PREFIX)/include
++	chmod 644 $(DESTDIR)$(PREFIX)/include/mcpp_out.h
```

Comment:
Instead of `cp` then `chmod`, use `install -m644` :)

Reply:
And static libraries don't have to be 755, they can be 644 just fine.

Reply:
I really should... intent was to be cautious if I PR the patch upstream.
Original uses cp so wasn't sure if this was supposed to be able to work on jank environments w/o install, or if this makefile just makes no sense.

Reply:
Done in future push.